### ISSUE TITLE
[BUG]: Python always results in compiler error

### ISSUE THREAD

**Issue by FelixDombek-TomTom** [compiler-explorer/compiler-explorer#4276]
### Describe the bug

All available Python versions result in an error `r is undefined` for any valid program.

### Steps to reproduce

1. select any Python compiler
2. write `x = 1` or `import re` or whatever you want

### Expected behavior

The program is compiled

### Reproduction link

https://godbolt.org/z/4bzzKczGd

### Screenshots

Not applicable

**Comment by jeremy-rifkin** [comment]
Thanks for the report, in trying to reproduce locally I can't even open the output window 😅 We'll take a look!

**Comment by partouf** [comment]
The error happens in the frontend, but it looks like the json is sort of incorrect.

**Comment by jeremy-rifkin** [comment]
This does appear to be a frontend issue and for some reason I can't reproduce locally

**Comment by jeremy-rifkin** [comment]
On the site if I reset code & UI layout, change to python, and type `x = 1` everything works. Somehow some pane must have gotten into a weird state and taking your repro link locally I can reproduce the bug. Somehow the backend is sending a `result.execResult.buildResult` that doesn't match the `BuildResult` type and doesn't have an stdout/stderr.

**Comment by RubenRBS** [comment]
Just got bitten by this coming from a clean state and trying to execute Python some code with the output pane open.

As @jeremy-rifkin discovered, it's because `result.execResult.buildResult` type seems to be plain wrong, which is my fault, but then I don't understand what type it's supposed to be!

If anyone wants to know, the `buildResult` only has a `code: 0`, and the stacktrace is:
```
TypeError: r is undefined
    addOutputLines output.ts:174
    onCompileResult output.ts:218
    emit goldenlayout.js:255
    emit event-hub.ts:69
    handleCompileRequestAndResult compiler.ts:1666
    onCompileResponse compiler.ts:1678
    sendCompile compiler.ts:1376
```

**Comment by RubenRBS** [comment]
Hi, this has now been fixed by #4289 and is now live, thanks for your patience!
